cpp/tests/io/parquet_chunked_reader_test.cu

```diff
@@ -1886,6 +1891,51 @@ TEST_F(ParquetChunkedReaderTest, TestNumRowsPerSourceEmptyTable)
     std::equal(expected_counts.cbegin(), expected_counts.cend(), num_rows_per_source.cbegin()));
 }
 
+TEST_F(ParquetReaderTest, ManyLargeLists)
+{
+  auto const stream = cudf::get_default_stream();
+
+  // Generate a list<bool> column with 10M outer rows and 2 bools per row
```

Comment:
I don't think the exact numbers should be in the comments. Easy to become outdated. Same for 150x below.